main.tf

```
@@ -88,11 +88,10 @@ resource "aws_cloudwatch_event_rule" "this" {
   event_bus_name = var.create_bus ? aws_cloudwatch_event_bus.this[0].name : var.bus_name
 
   description         = lookup(each.value, "description", null)
-  is_enabled          = lookup(each.value, "enabled", null)
   event_pattern       = lookup(each.value, "event_pattern", null)
   schedule_expression = lookup(each.value, "schedule_expression", null)
   role_arn            = lookup(each.value, "role_arn", false) ? aws_iam_role.eventbridge[0].arn : null
-  state               = lookup(each.value, "state", null)
+  state               = coalesce(lookup(each.value, "enabled", null), false) ? "ENABLED" : "DISABLED"
```

```suggestion
  state               = try(each.value.enabled ? "ENABLED" : "DISABLED", tobool(each.value.state) ? "ENABLED" : "DISABLED", "DISABLED")
```
How about this? It should handle previous cases (with "enabled" and "state") and be backward-compatible.

---

Will this work?

---

Hey @antonbabenko Yes that works, and deploys fine. 

However, I now get the following error when running the prehook:

❯ pre-commit run -a
Terraform fmt............................................................Passed
Terraform docs...........................................................Failed
- hook id: terraform_docs
- files were modified by this hook
Terraform validate with tflint...........................................Passed
Terraform validate.......................................................Passed
check for merge conflicts................................................Passed
fix end of files.........................................................Passed
trim trailing whitespace.................................................Passed